x/ibc/02-client/keeper/keeper.go

```diff
@@ -141,16 +143,25 @@ func (k Keeper) GetAllClients(ctx sdk.Context) (states []exported.ClientState) {
 // https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#example-implementation
 func (k Keeper) initialize(
 	ctx sdk.Context, clientID string, clientType exported.ClientType,
-	consensusState exported.ConsensusState,
+	consensusState exported.ConsensusState, trustingPeriod, unbondingPeriod time.Duration,
 ) (exported.ClientState, error) {
-	var clientState exported.ClientState
 	height := uint64(ctx.BlockHeight())
```

Comment:
Should not be using our own chain's BlockHeight here. Should be using the other chain's height.

Fixed this on my branch by adding Height field and `GetHeight` method to `ConsensusState`